src/frontend/components/actions/edit.tsx

```diff
@@ -41,15 +42,20 @@ const Edit: FC<ActionProps> = (props) => {
     return false
   }
 
+  const contentTag = getActionElementCss(resource, action, 'drawer-content')
+  const formTag = getActionElementCss(resource, action, 'form')
+  const footerTag = getActionElementCss(resource, action, 'drawer-footer')
```

Comment:
As I said, unify naming. You use `content` here:
https://github.com/SoftwareBrothers/adminjs/pull/1268/files#diff-33aa50ef73bc5baf6e4a148d08aa9adf1d949ab9f82674ca4e72e4da7264c623R69
and in this component it's `drawer-content`. I think `drawer-content` is a better name since it matches the component